packages/cli/src/bin.ts

```diff
@@ -1,7 +1,12 @@
 #!/usr/bin/env node
 
 import './tools/gracefulifyFs';
+import semver from 'semver';
+import chalk from 'chalk';
 
-import {run} from './';
-
-run();
+if (semver.satisfies(process.version, '>=14')) {
+  const {run} = require('./');
+  run();
+} else {
+  console.error(`${chalk.red('You need at least Node 14 to run CLI.')}`);
```

Comment:
How about something like this:
> React Native needs Node.js >=14. You're currently on version 12.3.4. Please upgrade Node.js to a supported version and try again.
```suggestion
  console.error(`${chalk.red('React Native needs Node.js ${nodeVersionRange}. You're currently on version ${process.version}. Please upgrade Node.js to a supported version and try again.')}`);
```

Reply:
Looks good. Updated 😄